### PR TITLE
Add hidden text for second conditional question

### PR DESCRIPTION
**Pull request by alexwllms**
@benlovell @jongilbraith

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -24,10 +24,11 @@ en:
     heading: Register your firm
     introduction: Register your firm on our Retirement Adviser Directory
     button_next: Next
-    
+
```
Comment (benlovell) [inline config/locales/en.yml]: :heart_eyes:

**Comment by benlovell** [comment]
:ship:

**Comment by benlovell** [comment]
Although, for folks with JS disabled this might be a little misleading.

**Comment by alexwllms** [comment]
Good point. Perhaps worth asking DAC for advice on this one.

**Comment by alexwllms** [comment]
Thumb and merge please @jongilbraith

**Comment by jongilbraith** [comment]
:+1:

**Pull request merged**